zipkin-client/src/main/java/com/smoketurner/dropwizard/zipkin/client/ZipkinClientBuilder.java

```diff
@@ -16,33 +16,57 @@
 package com.smoketurner.dropwizard.zipkin.client;
 
 import brave.http.HttpTracing;
+import brave.httpclient.TracingHttpClientBuilder;
 import brave.jaxrs2.TracingClientFilter;
+import com.google.common.base.Strings;
+import io.dropwizard.client.HttpClientBuilder;
 import io.dropwizard.client.JerseyClientBuilder;
+import io.dropwizard.client.JerseyClientConfiguration;
 import io.dropwizard.setup.Environment;
-import java.util.Objects;
 import javax.ws.rs.client.Client;
 
-public class ZipkinClientBuilder {
+public class ZipkinClientBuilder extends JerseyClientBuilder {
   private final Environment environment;
-  private final HttpTracing tracing;
+  private HttpTracing httpTracing;
 
   /**
    * Constructor
    *
    * @param environment Environment
-   * @param tracing HttpTracing instance
+   * @param httpTracing HttpTracing instance
    */
-  public ZipkinClientBuilder(final Environment environment, final HttpTracing tracing) {
-    this.environment = Objects.requireNonNull(environment);
-    this.tracing = Objects.requireNonNull(tracing);
+  public ZipkinClientBuilder(final Environment environment, final HttpTracing httpTracing) {
```

Comment:
another way is to deprecate the old type and call the new one ZipkinJerseyClientBuilder